src/components/Datagrid/headerIcons.ts

```diff
@@ -0,0 +1,18 @@
+import { SpriteMap } from "@glideapps/glide-data-grid";
+
+export const headerIcons: SpriteMap = {
+  arrowUp:
+    p => `<svg width="20" height="20"  viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
**question**: can those icons be in MacawUI instead?

Reply:
It's not possible because datagrid due to type incompatibility